cmd/node/node.go

```diff
@@ -1021,11 +1012,27 @@ func (app *SpacemeshApp) Start(*cobra.Command, []string) {
 		defer srv.Shutdown(ctx)
 		go func() {
 			if err := srv.ListenAndServe(); err != nil {
-				logger.With().Error("cannot start http server", log.Err(err))
+				logger.With().Error("cannot start pprof http server", log.Err(err))
 			}
 		}()
 	}
 
+	if app.Config.ProfilerUrl != "" {
+		p, err := profiler.Start(profiler.Config{
+			ApplicationName: "go-spacemesh",
```

Comment:
@y0sher this should be provided via a CLI flag. In spacecraft we have multiple miners so that we can distinguish the profiler output for each miner differently.